core/include/traccc/seeding/detail/seeding_config.hpp

```diff
@@ -114,7 +114,9 @@ struct seedfinder_config {
 
     TRACCC_HOST_DEVICE
     unsigned int get_max_neighbor_bins() const {
-        return std::pow(neighbor_scope[0] + neighbor_scope[1] + 1, 2);
+        unsigned int t = neighbor_scope[0] + neighbor_scope[1] + 1;
+        return t * t;
+            std::pow(neighbor_scope[0] + neighbor_scope[1] + 1, 2));
```

Comment:
:frowning: This line was left in by mistake. (Stephen didn't set up his comment to be applied as-is.)